README.md

```diff
@@ -55,12 +55,22 @@ Here's a *snapshot* of the script you need (*at this point in the exercise*): [g
 
 ### 3. Set the `TO_ADDRESS` in the Script
 
-In the editor window you should expect to see:
+**_Warning:_** If you do not uncomment and set your email as the value of
+`TO_ADDRESS`, it is possible for someone who has web skills to alter your form
+and send emailed data to an arbitrary email address.
 
-![3-script-editor-showing-script](https://cloud.githubusercontent.com/assets/194400/10560379/9efa5b3a-7501-11e5-96ba-a9e3b2d77ee4.png)
+This risk may not be very likely. Instead, if you wish, you can leave this
+variable commented out if you accept this possible risk but want the added
+convenience of setting this email variable inside your HTML form as a
+`data-email` attribute. This allows you to easily change where to send emails
```

Comment:
this can be used to send spam email from a legitimate GMail account. it's not as _unlikely_ as we would hope ... 🙄